frontend-resources/pom.xml

```diff
@@ -23,7 +23,7 @@
 	<build>
 		<resources>
 			<resource>
-				<directory>${workingDirectory}/app/dist/</directory>
+				<directory>${workingDirectory}/build/</directory>
```

Comment:
In #1120  you mentioned  a change of the compile output directory. The change here does not exactly mirror this. Is that correct anyways?

Reply:
Yes, It was actually even under `/app/...`, didn't mention that above